src/data/faq_de.json

```diff
@@ -35,17 +35,17 @@
                 "title": "Häufige Probleme",
                 "id": "most_asked",
                 "accordion": [
-                    {
-                        "title": "URSACHE: 2001, trying to e establish a secure connection to the server",
-                        "anchor": "cause2001",
-                        "active": true,
-                        "textblock": [
+                    {
+                        "title": "URSACHE: 2001, trying to establish a secure connection to the server",
+                        "anchor": "cause2001",
+                        "active": true,
+                        "textblock": [
                             "Das Sicherheitszertifikat <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b> ist möglicherweise auf Ihrem Gerät deaktiviert. Dies ist eine bekannte Ursache, um folgende Benachrichtigung auszulösen:",
-                            "<b>'URSACHE: 2001 Etwas ist schief gelaufen. An error occurred while trying to establish a secure connection to the server.'</b>",

```

Comment:
Again, did not change anything here, don't know why this is happening

Reply:
@Ein-Tim 
> Again, did not change anything here, don't know why this is happening

Your editor is changing whitespace. If you go to "Files changed", click on the gear-wheel, then select "Hide whitespace changes" , Apply and reload, you will see that the unexpected changes disappear from the view.